packages/scripts/scripts/env/install.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+const commandExistsSync = require( 'command-exists' ).sync;
```

Comment:
We can make this look a little closer to `import` syntax.

```suggestion
const { sync: commandExistsSync } = require( 'command-exists' );
```